_posts/2017-01-16-Jak-psat-php-Uloziste-kodu+databaze+frameworky.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: Jak psát PHP? (3/3) - Úložiště kódu, databáze a frameworky
```

Comment:
Až teď mi došlo, že ten titulek je zavádějící. U jak psát PHP bych čekal krok za krokem jako na jakpsatweb.cz

V první větě čtu "V poslední části přehledu o nástrojích" a to mi dává smysl. Je to přehled nástrojů. Takový titulek by mi seděl!